### PR TITLE
feat: add dashboard event api, liv-9542

### PR DESCRIPTION
**Pull request by Michaelvilleneuve**

### CODE REVIEW

File: dist/api/dashboard/event.d.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import type { DashboardEvent } from '../../types/dashboard/event';
```
Comment (jinevo) [inline dist/api/dashboard/event.d.ts:1]: The `dashboard` folder is obsolete?

File: src/api/tracking/event.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+/**
+ * 
+ * Store a value under a specific key. This storage is persistant and shared across participants of the event
+ * 
+ * @example Tracking.Event.track({
+ *   key: 'emoji-reactions',
+ *   value: 'joy'
+ * })
+ *
+ * @doc https://developers.livestorm.co/docs/storage#setitem
```
Comment (jinevo) [inline src/api/tracking/event.ts:12]: Not updated

File: src/api/tracking/event.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import Configuration from '@/configuration'
+import type { TrackingEvent } from '@/types/tracking/event'
+/**
+ * 
+ * Store a value under a specific key. This storage is persistant and shared across participants of the event
```
Comment (jinevo) [inline src/api/tracking/event.ts:5]: Not updated

File: dist/api/dashboard/event.d.ts (hunk 1 above)
Comment (Michaelvilleneuve) [reply dist/api/dashboard/event.d.ts:1]: 👍

File: src/api/tracking/event.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ * 
+ * @example Tracking.Event.track({
+ *   key: 'emoji-reactions',
+ *   value: 'joy'
+ * })
+ *
+ * @doc https://developers.livestorm.co/docs/tracking#track
+ * 
+ */
+export async function track(options: TrackingEvent): Promise<Response> {
```
Comment (jinevo) [inline src/api/tracking/event.ts:15]:
```suggestion
export async function tracktrack<T extends string | number | Record<string, unknown>>(options: TrackingEvent<T>): Promise<Response> {
```

File: src/types/tracking/event.ts [hunk 5]
```diff
@@ -0,0 +1,4 @@
+export type TrackingEvent = {
```
Comment (jinevo) [inline src/types/tracking/event.ts:1]:
```suggestion
export type TrackingEvent<T> = {
```

File: src/types/tracking/event.ts [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+    key: string;
+    value?: string | number | Record<string, unknown>;
```
Comment (jinevo) [inline src/types/tracking/event.ts:3]:
```suggestion
    value?: T;
```

**Review by jinevo — APPROVED** [review]

**Pull request merged**